### PR TITLE
adding validation to project settings view

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Resolves #505 
Resolves #507 
Resolves #508

**Comment by gvn** [comment]
Setting limit at 128 per standup discussion.

**Comment by gvn** [comment]
I also just realized that super long titles can break the Card layouts, so I need to fix that.

**Comment by gvn** [comment]
R? @k88hudson & @thisandagain

**Comment by k88hudson** [comment]
R+

File: src/components/card/card.less [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
     .text {
       flex: 1;
+      word-wrap: break-word;
```
Comment (thisandagain) [inline src/components/card/card.less:37]: `break-word` may still cause some issues. Particularly in languages like german where a single "word" may go beyond the width of the card. Any objections to using `ellipsis`?

Comment (k88hudson) [inline src/components/card/card.less:37]: yeah, that's a good point

**Comment by thisandagain** [comment]
One minor comment... otherwise R+

File: src/components/card/card.less (hunk 1 above)
Comment (gvn) [inline src/components/card/card.less:37]: Isn't it better to get the whole title in there though even if a word is split?

Comment (thisandagain) [inline src/components/card/card.less:37]: Yeah. That's true. Ok ... ship it. :shipit:

Comment (gvn) [inline src/components/card/card.less:37]: dance2

**Pull request merged**